app/views/tag/_tagging.html.erb

```diff
@@ -82,4 +85,25 @@ $(".label").on("click", function(e){
 </form>
 <% end %>
 
+<script>
+  $("#tags-open").click(function(){
+      $("#tagform").toggle();
+  });
+</script>
+
+<style>
+  #tagform{
+    display:none;
+  }
+</style>
+
+<style>
```

Comment:
Please shift all css in a single style tag. This will make the page little lighter in size.